cmd/group_helpers.go

```diff
@@ -142,3 +142,29 @@ func printGroupsOut(cmd *cobra.Command, groups ...*gitlab.Group) {
 	}
 	table.Render()
 }
+
+// printGroupProjectsOut prints the group project list/get commands to a table view or json
+func printGroupProjectsOut(cmd *cobra.Command, projects ...*gitlab.Project) {
```

Comment:
I think this function can also be used when we start creating the project commands since it accepts a `gitlab.Project`.

Can we rename this to `printProjectsOut`? Put this function in a file named `project_helpers.go` as well.

Reply:
@depwaswho94 you can remove this function now.